asserts/model.go

```diff
@@ -781,6 +781,25 @@ func (mod *Model) checkConsistency(db RODatabase, acck *AccountKey) error {
 	return nil
 }
 
+// NeedsKernelSetup tells if we need a kernel drivers tree for this model.
+func (mod *Model) NeedsKernelSetup() bool {
```

Comment:
I wouldn't expect this kind of decision in the assert package, I think it would be best to make it a helper in the kernel package instead which has other related bits anyway

Reply:
Moved to the kernel package now